client/src/app/pages/assessment/components/assessment-page-header.tsx

```diff
@@ -19,38 +20,38 @@ export const AssessmentPageHeader: React.FC<AssessmentPageHeaderProps> = ({
 }) => {
   const { t } = useTranslation();
   const history = useHistory();
+  const isArchetype = location.pathname.includes("/archetypes/");
```

Comment:
This has been repeated enough times that it is a good candidate to create a hook for....

```js
const isArchetype = useIsArchetypeFromLocation();
```

Reply:
All good ideas. Will work on refactoring.